addons/hr_presence/views/hr_employee_views.xml

```diff
@@ -38,13 +38,13 @@
             </xpath>
             <xpath expr="//div[hasclass('oe_kanban_details')]" position="inside">
                 <div class="oe_kanban_content">
-                    <div class="oe_kanban_row">
+                    <div class="oe_kanban_row d-flex gap-1">
```

Comment:
Good, thanks 👍 
To improve even further let's remove `mt-8` from the buttons and apply a `mb-[x]` to the `oe_kanban_row` or, even better,  applying a `gap-1` to `oe_kanban_content` instead... I'll let you decide according to the visual result on screen ;)

Since we're here, I propose to update this design to `milk`, meaning convert all the buttons to `secondary` and, eventually, use `success` for "Set as present" and `warning` for "Set as absent"